packages/firestore/test/integration/api/database.test.ts

```diff
@@ -1073,7 +1073,7 @@ apiDescribe('Database', (persistence: boolean) => {
   it('can start a new instance after shut down', async () => {
     return withTestDoc(persistence, async docRef => {
       const firestore = docRef.firestore;
-      await firestore.INTERNAL.shutdown();
+      await (firestore as any)._shutdown();
```

Comment:
`any` is disallowed by the linter (again, running `yarn lint` from packages/firestore will complain).  So you'll need a lint suppression. I'd recommend trying not to repeat it over and over.  So you could create a helper like Brian did for clearPersistence(): https://github.com/firebase/firebase-js-sdk/commit/7a15e7e54bbd80d04311865c46588b68eff93d3b#diff-1422c3d56f022a40f00b40d264351ffdL109

Reply:
Done.